news/14666-default-env

```diff
@@ -0,0 +1,20 @@
+### Enhancements
+
+* Add `context.default_activation_env` setting, which allows users to customize which environment should get auto activated on initialization (if `context.auto_activate` is enabled), as well as `conda activate` calls without arguments. Defaults to `base`. (#14666)
+
+### Bug fixes
+
+* <news item>
+
+### Deprecations
+
+* Mark `context.auto_activate_base` as pending deprecation in favor of `context.auto_activate`.
+  It will be permanently removed in 26.1. (#14666)
```

Comment:
We typically haven't included the actual removal version in the changelog, is this something we should start including?

```suggestion
* Mark `conda.base.context.context.auto_activate_base` as pending deprecation. Use `conda.base.context.context.auto_activate` instead.  (#14666)
```

Reply:
I think it's useful to put in the notes, yes.

Reply:
```suggestion
* Mark `conda.base.context.context.auto_activate_base` as pending deprecation. Use `conda.base.context.context.auto_activate` instead. It will be permanently removed in 26.3. (#14666)
```